templates/company/about.html

```diff
@@ -0,0 +1,464 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/19TIwT-GU5wADQmohicj14MnpHEqNqp0422WLunFsbJU
+{% endblock meta_copydoc %}
+
+{% block title %}Canonical - company information{% endblock %}
+
+{% block canonical_url %}{{ request.host_url }}company/about{% endblock %}
+
+{% block meta_description %}
+  About Canonical |The publisher of Ubuntu and provider of open source security, support and services
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <div class="u-fixed-width">
+    <nav class="p-breadcrumbs" aria-label="Breadcrumbs">
+      <ol class="p-breadcrumbs__items p-text--x-small-capitalised">
+        <li class="p-breadcrumbs__item ">
+          <a href="/company">Company</a>
+        </li>
+        <li class="p-breadcrumbs__item ">About Canonical</li>
+      </ol>
+    </nav>
+    <hr class="p-rule" />
+  </div>
+
+  <section class="p-section--hero">
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <h1 class="u-no-margin--bottom">About Canonical</h1>
+        <h2>
```

Comment:
I'm not sure that this is supposed to actually be a heading, I think it's just supposed to be styled as one